File: sample-app/src/main/java/io/github/lazymockbean/sampleapp/service/SampleServiceA.java
package io.github.lazymockbean.sampleapp.service;

import org.springframework.stereotype.Service;

@Service
public class SampleServiceA {
    public String getSample1() {
        return "sample-A-1";
    }

    public String getSample2() {
        return "sample-A-2";
    }
}

File: sample-app/src/main/kotlin/com/taesk/lazymockbean/sampleapp/service/SampleServiceA.kt
package com.taesk.lazymockbean.sampleapp.service

import org.springframework.stereotype.Service

@Service
class SampleServiceA {
    fun getSample1(): String {
        return "sample-A-1"
    }

    fun getSample2(): String {
        return "sample-A-2"
    }
}
